paper.tex

```diff
@@ -68,28 +68,62 @@ \section*{Ancestral graphs and stochastic processes}
 % See https://github.com/tskit-dev/what-is-an-arg-paper/discussions/13
 % for discussions on this section
 
-The coalescent
-process~\citep{kingman1982coalescent,kingman1982genealogy,hudson1983testing,
-tajima1983evolutionary}
-
-The Ancestral Recombination Graph (ARG) was introduced by
-Griffiths~\citep{griffiths1991two,griffiths1997ancestral}.
-The ``big'' ARG~\citep{ethier1990two},
-and the ``little ARG'' traversed by
-Hudson's algorithm~\citep{hudson1983properties}.
-
+The simplest prominent example of genetic ancestry generated by a stochastic process
+is the coalescent process \citep{kingman1982coalescent,kingman1982genealogy,
+hudson1983testing, tajima1983evolutionary}, which describes the random
+ancestral tree at one non-recombining locus obtained by sampling a relatively
+small number of individuals at random from a large population.
+
+In a recombining organism, the genetic ancestry of a longer segment of DNA is 
+no longer described by a single tree. Instead, ancestral relationships in a
+sample can be represented by a sequence of correlated, marginal trees along the
+genome. The Ancestral Recombination Graph (ARG) was introduced by
+Griffiths~\citep{griffiths1991two,griffiths1997ancestral} as a branching-coalescing 
+stochastic process whose realisations are graphs which into which all of the
+marginal ancestral trees are embedded. Coalescences encode common ancestry
+as they do in the coalescent, while branching events encode points at which a
+marginal tree changes from one position along the genome to the next.
+
+There are several formulations of the ARG, which vary by
+the amount of extra information they capture beyond a minimal representation
+of all marginal trees. Two prominent examples are the ``big''
+ARG~\citep{ethier1990two}, which has a particularly simple specification as a
+stochastic process but is computationally infeasible, and the ``little ARG''
+traversed by Hudson's algorithm~\citep{hudson1983properties}, which
```

Comment:
Didn't Griffiths work with the little ARG too?

Reply:
I'm beginning to think I'm not 100% clear on the details of what "big" and "little" ARGs mean. I think we should definitely have a quick summary.

Reply:
I agree - this is something we'll have to tease out more carefully as it's all very confusing. We don't need to do it here though, let's get this much down on paper first.

Reply:
I agree - let's put in a comment to that effect in the latex and then merge this.